detection-rules/attachment_html_smuggling_atob.yml

```
@@ -1,22 +1,29 @@
 name: "Attachment: HTML smuggling with atob and high entropy"
 description: |
   Recursively scans files and archives to detect HTML smuggling techniques.
+references:
+  - "https://delivr.to/payloads?id=7dbf0d83-1557-4345-bf67-d18c4256b0c1"
 type: "rule"
 severity: "high"
 source: |
   type.inbound
-  and any(attachments, .size <= 60000 and
+  and any(attachments, .size <= 20000 and
```

```suggestion
  and any(attachments, .size <= 60000 and
```
Rule was recently updated to 60000 based on feedback from customers. Should probably keep it at that.